docs/src/app/components/pages/customization/themes.jsx

```diff
@@ -115,6 +116,32 @@ const ThemesPage = React.createClass({
         letterSpacing: '0',
         fontWeight: Typography.fontWeightMedium,
         color: Typography.textDarkBlack
+      },
+      liveExamplePaper: {
+        backgroundColor: canvasColor,
+        marginBottom: 32,
+        overflow: 'hidden'
+      },
+      liveExampleBlock: {
+        borderRadius: '0 0 2px 0',
+        padding: this.state.muiTheme.rawTheme.spacing.desktopGutter,
+        margin: 0,
+      },
+      headline: {
+        fontSize: '24px',
```

Comment:
Thanks for noticing that! I know that some properties do need the unit, so I'll hold off on that for now, and take my time to clean things up a little in the future ;)